services/QuillLMS/app/controllers/api/v1/feedback_histories_controller.rb

```diff
@@ -94,4 +94,12 @@ def destroy
       ]
     )[:feedback_histories]
   end
+
+  private def rename_activity_session_uid_params_key(params_hash)
+    # We've changed the name of the model param in FeedbackHistory, but don't (currently)
+    # want to go back and change every single Feedback API in our stack to use a new signature.
+    # This swap allows us to maintain the current API while making the new code work.
+    params_hash[:session_uid] = params_hash.delete(:activity_session_uid)
```

Comment:
What is the purpose of the `batch` method? Will it be used only for a one-time backfill, or continuously?

Reply:
`batch` is actually the primary call site for creating `FeedbackHistory` records.  The Go feedback function fans out to a bunch of APIs to get feedback, and then bundles them up into one `batch` call.  So this is used all the time.